### ISSUE TITLE
eslint require error

### ISSUE THREAD

**Issue by raiscui** [MiguelCastillo/Brackets-InteractiveLinter#165]
when use
`"interactive-linter.javascript": ["eslint"],
    "language": {
        "javascript": {
            "linting.prefer": [
                "ESLint"
            ],
            "linting.usePreferredOnly": true
        }
    }`

and `.eslintrc`
`  "parser": "babel-eslint",`

the InteractiveLinter will echo `Interactive Linter: 1 Linter Problem`
like that:
`
Module name "babel-eslint" has not been loaded yet for context: _. Use require([]) http://requirejs.org/docs/errors.html#notloaded	[undefined] - undefined
`

**Comment by MiguelCastillo** [comment]
@raiscui yeah configuring anything but linting rules is supported.  Currently, supporting the parser field isn't a priority as that's not quite as common of a use case.  However, I will happily accept a PR :)
